### ISSUE TITLE
Redstone Integrator does not output redstone signal

### ISSUE THREAD

**Issue by djmattyg007** [Seniorendi/AdvancedPeripherals#265]
### Descripe

The Redstone Integrator does not output any signal, despite explicitly turning it on in the lua REPL. Redstone dust is not activated, and redstone lamps do not light up.

Please note that I've selected Version `1.16.5-0.7.7r` only because `1.16.5-0.7.7.1r` isn't available to be picked. I am using the very latest version for Minecraft 1.16.5.

### Steps to reproduce

1. Attach a Redstone Integrator to an Advanced Computer using wired modems.
2. Place a redstone lamp, connected to any side of the Redstone Integrator with redstone dust.
3. In the lua REPL on the computer, run the following code:
   ```
   rsi = peripheral.wrap("redstoneIntegrator_0")
   rsi.setOutput(<insert redstone dust side here>, true)
   ```
4. Observe that the lamp does not turn on.

### Multiplayer?

No

### Version

1.16.5-0.7.7.1r

### Minecraft, Forge and maybe other related mods versions

Minecraft 1.16.5 Forge 36.2.26 CC: Tweaked 1.100.2 JEI 7.7.1.145

**Comment by SirEndii** [comment]
0.7.7r should not even work due to the reason that it does not contain any class files.
Are you sure you play with this version?

**Comment by djmattyg007** [comment]
As I stated near the top of my message:

> Please note that I've selected Version 1.16.5-0.7.7r only because 1.16.5-0.7.7.1r isn't available to be picked. I am using the very latest version for Minecraft 1.16.5.

I wasn't able to pick 0.7.7.1r when filling out the form.

**Comment by SirEndii** [comment]
Sorry, I misunderstood that

**Comment by SirEndii** [comment]
I tested it a bit. I can approve that the redstone integrator does not work in 1.16. But it works in 1.18.

**Comment by SirEndii** [comment]
If you want, you can use this version.
[advancedperipherals-1.16.5-0.7.7.1r.zip](https://github.com/Seniorendi/AdvancedPeripherals/files/8020901/advancedperipherals-1.16.5-0.7.7.1r.zip)
Unzip the file before using.

**Comment by djmattyg007** [comment]
To confirm, despite having the same version number, the code is different?

**Comment by SirEndii** [comment]
yes it's
I just didn't changed the version in the properties file(I only do that when I do a update)

**Comment by djmattyg007** [comment]
I've finally had a chance to try your patched version, and can confirm that it works 👍

**Comment by SirEndii** [comment]
> I've finally had a chance to try your patched version, and can confirm that it works 👍

I finally had a chance to prepare the release 😓 
Releasing later today

**Comment by SirEndii** [comment]
Fixed in 0.7.8r
